From 48e8f353ea4fb6fa32d58ae5c1faac69cf83c942 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 16:37:27 +0700
Subject: [PATCH 691/909] Revert "arm64: dts: renesas: r8a7796: Connect
 Ethernet-AVB to IPMMU-DS0"

commit dc410e682aec533c0acd8fc8d763d7f6b0e1794d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 00f9a5180a3ddc31ce4c0655a730bf14d4e35262
to keep IPMMU address translation for Ethernet-AVB is disabled
by default, maintain the same IPMMU support environment with
previous released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index c35302321e34..f17be609c3de 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -1566,7 +1566,6 @@
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 812>;
 			phy-mode = "rgmii";
-			iommus = <&ipmmu_ds0 16>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 			status = "disabled";
-- 
2.17.1

